yash-cli/tests/scripted_test/return-y.sh

```
@@ -10,7 +10,7 @@ f() { return --no-return 3; }
 f
 __IN__
 
-test_OE -e 3 'short option name still accepted under the portable option' -o portable
+test_O -d -e 2 'short option name also rejected under the portable option' -o portable
 f() { return -n 3; }
```

Shellcheck (0.11.0)

[error] Only one integer 0-255 can be returned. Use stdout for other data.

(SC2151)